### PR TITLE
Get tags

### PR DESCRIPTION
**Pull request by nkottary**

### CODE REVIEW

**Comment by christopher-dG** [comment]
Could you add GitLab support too, please? [Docs here.](https://docs.gitlab.com/ce/api/tags.html#list-project-repository-tags)

File: src/forges/GitLab/tags.jl [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    tag_name::String
+    description::String
+end
+
+@json struct Tag
+    commit::Commit
+    release::Release
+    name::String
+    target::String
+    message::Union{Nothing, String}
```
Comment (christopher-dG) [inline src/forges/GitLab/tags.jl]:
```suggestion
    message::String
```
The field automatically becomes nullable via the `@json` macro.

File: src/forges/GitLab/tags.jl [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+end
+
+endpoint(::GitLabAPI, ::typeof(get_tags), project::Integer) =
```
Comment (christopher-dG) [inline src/forges/GitLab/tags.jl]: You can also do the `owner::AStr, repo::AStr` form here.

Comment (nkottary) [reply src/forges/GitLab/tags.jl]: Does this work? Ex: https://gitlab.com/api/v4/projects/12394864/repository/tags works but not https://gitlab.com/api/v4/projects/nkottary/testreg/repository/tags

Comment (christopher-dG) [reply src/forges/GitLab/tags.jl]: The `/` in `owner/project` needs to be escaped, e.g. https://gitlab.com/api/v4/projects/nkottary%2Ftestreg/repository/tags. `GitLab.encode` does that for us.

**Pull request merged**